packages/cf-component-card/src/CardContent.js

```diff
@@ -16,7 +17,7 @@ class CardContent extends React.Component {
         {this.props.children}
         { this.props.footerMessage ? (
           <div className="cf-card__footer_message">
```

Comment:
Just use <Text> directly, no need to wrap it just so we can apply a CSS class with 1 property.